botorch/optim/optimize.py

```diff
@@ -446,7 +446,8 @@ def optimize_acqf(
         inequality_constraints: A list of tuples (indices, coefficients, rhs),
             with each tuple encoding an inequality constraint of the form
             `\sum_i (X[indices[i]] * coefficients[i]) >= rhs`. `indices` and
-            `coefficients` should be torch tensors. When q=1, or when
+            `coefficients` should be torch tensors. See the docstring of
+            `make_scipy_linear_constraints` for an example. When q=1, or when
```

Comment:
This is definitely an improvement but what is `make_scipy_linear_constraints` and where does one find it? It is not imported in this file, so the user would have to search the repo to find where it is defined.

Reply:
google botorch make_scipy_linear_constraints. Also, look at [this post](https://github.com/pytorch/botorch/discussions/1899) I made when trying to understand how it works

Reply:
Fair point, this could definitely improved upon, but I wanted to get something out that's actionable (search/googleable) right away rather than putting this on a backlog list and then forget about it :)

Reply:
Adding the 
>see the docstring of `make_scipy_linear_constraints` for an example 

allows for googling

I just searched for it and the 3rd option was the charm, giving me this [link](https://botorch.org/v/0.6.1/api/optim.html), where you can find, with a bit more search, [this part of the page](https://botorch.org/v/0.6.1/api/optim.html#botorch.optim.parameter_constraints.make_scipy_linear_constraints)
![image](https://github.com/pytorch/botorch/assets/57282198/fb0d7caf-80ca-4a00-a6f5-914c370235b4)


I can see it's not the best option, but at least it allows the user to understand what's going on behind the scenes.

Reply:
You could also link to [this question I made in stackoverflow](https://stackoverflow.com/questions/76523226/botorch-equality-constraints-argument-example-in-the-optimize-acqf-function) in the docs. IT's just my rambling and understanding of how the process is working, but could be useful